cmd/cmd.go

```diff
@@ -732,6 +820,44 @@ func RunServer(cmd *cobra.Command, _ []string) error {
 	return server.Serve(ln, origins)
 }
 
+func getBase64Image(filePath string) string {
+	if _, err := os.Stat(filePath); os.IsNotExist(err) { // If the file does not exist
+		return ""
+	}
+
+	file, err := os.Open(filePath)
+	if err != nil {
+		fmt.Println("Error:", err)
+		return ""
+	}
+	defer file.Close()
+
+	data, err := io.ReadAll(file)
+	if err != nil {
+		fmt.Println("Error:", err)
+		return ""
+	}
+
+	contentType := http.DetectContentType(data)
+	allowedTypes := []string{"image/jpeg", "image/jpg", "image/svg+xml", "image/png"}
+	if !contains(allowedTypes, contentType) {
+		return ""
+	}
+
+	// Convert the image data to base64
+	imgBase64 := base64.StdEncoding.EncodeToString(data)
+	return imgBase64
+}
+
+func contains(s []string, e string) bool {
```

Comment:
I'd prefer having this `contains` check just in the code rather than the help function personally